server/pypi/packages/scipy/patches/chaquopy.patch

```diff
@@ -274,11 +192,11 @@ diff -aur src-orig/scipy/sparse/linalg/_propack/setup.py src/scipy/sparse/linalg
 diff -aur src-orig/scipy/special/ellint_carlson_cpp_lite/ellint_arith_aux.hh src/scipy/special/ellint_carlson_cpp_lite/ellint_arith_aux.hh
 --- src-orig/scipy/special/ellint_carlson_cpp_lite/ellint_arith_aux.hh	2023-09-20 11:08:03.696956933 +0200
 +++ src/scipy/special/ellint_carlson_cpp_lite/ellint_arith_aux.hh	2023-09-21 13:13:09.439909816 +0200
-@@ -4,7 +4,7 @@
- 
+@@ -5,6 +5,8 @@
  #include <cstddef>
  #include "ellint_typing.hh"
--
+
++// Chaquopy: added <algorithm>
 +#include <algorithm>
```

Comment:
All edits should include a "Chaquopy" marker so it's obvious they weren't part of the original source